app/shared/design-system/components/primitives/numeric-gesture-control.ts

```diff
@@ -28,6 +28,14 @@ type Options = {
   }) => void;
 };
 
+type State = {
+  value: number;
+  cursor: SVGElement | null;
```

Comment:
Nit: I generally prefer undefined when possible, because it works with optional arguments in js